extra-utils/pocketsphinx/spec

```diff
@@ -1,4 +1,5 @@
-VER=5prealpha 
+VER=5prealpha
```

Comment:
I think this VER doesn't comply with Styling Manual, but I have no idea how to redefine it ......